package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
The problem (as I mentioned earlier) is in next sentence of the doc: "It is still your program's responsibility to handle the lack of the dependency..." It means, that in patternfly-react codebase, we'd have to explicitly check if react-bootstrap is available and handle the case when it isn't.